_posts/2022-01-14-quorn-a-story-about-single-cell-protein.md

```diff
@@ -0,0 +1,197 @@
+
+Alternatively, if corn ethanol gets banned due to policy changes, we're going to have a lot of un-used corn milling and hydrolysing infrastructure. These could easily be adapted into Quorn-type protein production plants. 
+
+**Taste and Texture Matter**
+
+A food products' **organoleptic properties** matter a lot in the meat substitute space. Quorn's success comes in part due to the fact the the hyphal product, after heat treatment and freezing, resembles the fibrous texture of meat. 
+
+![](/assets/images/quorn/quorn-meatless-pieces-fried-texture.png){:height="350" .center}
+*Fried Quorn nuggets - Source: veggiepattytastetest.com*
+
+
+##### Quorn as a Miso
+
+I'd love to get my hands on some *F. venenatum* biomass, centrifuged, **without heat-treating to remove RNA**. I would take this 20-30% solids biomass and heat treat it myself, keeping the 8-9% nucleic acids, and maybe make a miso or garum out of it, since it would contain plenty of natural 5'-ribonucleotides and glutamates. In theory it would make an umami bomb that doesn't have the flavour of yeast extracts. At the small amounts I would be consuming in either form, uric acid isn't a problem: so long as daily consumption is under the WHO recommendation of <2g RNA per day (a tablespoon of this stuff should contain around 0.3g)
+
+Maybe I'll ask Quorn for some, or make my own. The original patents for this bioprocess have already expired - though it would be difficult to replace Marlow Foods' 35 years of operating experience. 
+
+##### The Protein Crisis 
+
+The original goal at inception, to prevent protein malnutrition, was never really met - because thankfully, and generally speaking - the 'Protein Crisis' neve happened agricultural productivity kept up with population growth through the 1960s, 70s, 80s, and beyond. This was mostly due to the *Green Revolution* consisting of better crop strains, fertilizer use, and the like. SCP could never compete economically with plant proteins. 
+
+But I can't fault the authors of the original UN paper: in 1967 we hadn't even discovered RuBisCO, a key enzyme behind photosynthesis. This discovery wouldn't be made until well into the 1980s [12]. Before that, it might have been safe to assume that plants simply assimilate nitrogen to create protein, so we could bypass the process altogether by adding sugar and nitrogen into a bioreactor.
+
+As it turns out, protein synthesis and photosynthesis are directly coupled. Photosynthesis relies on the action of enzymes, especially RuBisCO, the massive enzyme used in carbon fixation and possibly the most abundant enzyme on earth. So yes, the nitrogen applied to soil is assimilated into protein, but that protein also enables photosynthesis and the formation of carbohydrates. It's impossible to compare Quorn to plant protein, especially since its primary feedstock - glucose - is derived from plants. 
```

Comment:
I'm also not sure about this paragraph. Is the summary something like "plants need proteins to photosynthesis, so lets eat those proteins"? Most (all?) of edible plant proteins are from seeds or seed-like parts of the plant, where there's not rubisco, but albumin, and in high concentration. Spinach isn't known for high protein content.

Reply:
You're right this is a confusing two paragraphs. I'm considering removing these two statements and ending it with "
SCP could never compete economically with plant proteins. "

The basic train of thought is this:

1) Prior to 1967, we may have assumed that plants produce fungi in a direct pathway of carbs+nitrogen--> protein
2) After we discovered RuBisCO and the full photosynthetic pathway, we realized it's more of a feedback loop: carbs+nitrogen --> protein, but some of that protein (as RuBisCO) enables the production of more carbs.

On re-reading this, it does seem like a weak / confusing statement. I think I'll remove it.

Reply:
OK - removed. "SCP could never compete economically with plant proteins." is a good enough stopping point without delving into details.